fluent-plugin-google-cloud.gemspec

```diff
@@ -19,14 +19,14 @@ eos
   gem.test_files    = gem.files.grep(/^(test)/)
   gem.require_paths = ['lib']
 
-  gem.add_runtime_dependency 'fluentd', '1.11.2'
+  gem.add_runtime_dependency 'fluentd', '~> 1.12', '>= 1.12.2', '< 1.13'
```

Comment:
Let me kick off an internal validation process for Fluentd `1.12.3`. As mentioned in https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/pull/447#issuecomment-810655681, the version has to pinned to an exact version for now, e.g.

`gem.add_runtime_dependency 'fluentd', '1.12.3'`

Please update.

Reply:
Okay, updated the PR to force exact version `1.12.3`. Ideally, can we move away from forcing fluentd to particular version. Otherwise, anytime we have to upgrade [KFO](https://github.com/vmware/kube-fluentd-operator) we will always have to fork this gem.